tests/README.md

```diff
@@ -14,7 +14,7 @@ The TodoMVC project has a great many implementations of exactly the same app usi
 
 ## Running the tests
 
-These tests use Selenium 2 (WebDriver), via the JavaScript API (WebdriverJS).  In order to run the tests you will need to install the dependencies. Run the following command from within the `tests` folder:
+These tests use Selenium 2 (WebDriver), via the JavaScript API (WebdriverJS).  In order to run the tests you will need to install the dependencies.  NPM to be version 2.0.0 or greater, so upgrade it first with `npm install -g npm` if `npm --version` outputs anything less than 2.0.0. Run the following command from within the `tests` folder:
```

Comment:
Updated to include must, the rest of the commands should be the same, so I didn't update where to run them.  Does it matter if they are run from anywhere or from the tests folder?  I think they do require being run from the tests folder.